lib/index.d.ts

```diff
@@ -66,7 +66,8 @@ declare module 'nact' {
     self: Ref<Msg>,
     name: ActorName,
     children: Map<ActorName, Ref<unknown>>,
-    log: Logger
+    log: Logger,
+    mailbox: Msg[]
```

Comment:
What should happen if the crash is escalated? Are the messages in the child dropped?

Reply:
Moved it out to an intermediate type between SupervisionContext and ActorContext to be shared with afterStop, as discussed.

Reply:
Added context to afterStop so that it can be handled there across children, as discussed.